packages/http-client/src/csrf-token-header.ts

```diff
@@ -126,7 +126,9 @@ function getResponseHeadersFromError(error: any): Record<string, any> {
 }
 
 function appendSlash<T extends HttpRequestConfigBase>(requestConfig: T): T {
-  if (!requestConfig.url!.endsWith('/')) {
+  if (!requestConfig.url) {
+    requestConfig.url = '/';
+  } else if (!requestConfig.url!.endsWith('/')) {
```

Comment:
[q] is the second `!` needed. You would expect that we know `requestConfig.url` is defined because of the first `if`statement.

Reply:
Yes, correct. I fixed it now.